src/content/blog/astro-520.mdx

```diff
@@ -0,0 +1,181 @@
+
+To enable trailing slash redirects, set the `trailingSlash` option in your `astro.config.mjs`:
+
+```js title="astro.config.mjs" ins={3}
+export default defineConfig({
+	adapter: node({ mode: 'standalone' }),
+	trailingSlash: 'never', // or 'always'
+});
+```
+
+Now, it doesn't matter whether your visitor navigates to `/about/`, `/about`, or even `/about///`. In production, they'll always end up on the correct page. For GET requests, the redirect will be a 301 (permanent) redirect, and for all other request methods, it will be a 308 (permanent, and preserve the request method) redirect.
+
+During development, Astro will display an error page instead of redirecting to help you catch any misconfigurations and bad links.
+
+For more information, see the [`trailingSlash` documentation](https://docs.astro.build/en/reference/configuration-reference/#trailingslash).
+
+## External redirects
+
+Astro now supports defining external redirects straight from the configuration. External redirects must start with `https` or `http`:
+
+```js
+import {defineConfig} from "astro/config"
+
+export default defineConfig({
+    redirects: {
+        "/about": "https://example.com/about",
+        "/news": {
+            status: 302,
+            destination: "https://example.com/news"
+        }
+    }
+})
+```
+
+
+## TOML frontmatter in Markdown
+
+Currently Astro supports YAML frontmatter in Markdown files. With this release, Astro now also supports TOML frontmatter in Markdown and MDX files. This is useful for adding existing content files with TOML frontmatter to your project from another framework such as Hugo, or you may just prefer TOML over YAML!
+
+No configuration is required to use TOML frontmatter in your content files: just use `+++` to delimit the frontmatter and Astro will parse it as TOML.
+
+```md ins={1-6}
++++
+date = 2025-01-30
+title = 'Use TOML frontmatter in Astro!'
+[params]
+  author = 'Houston'
++++
+
+# Support for TOML frontmatter is here!
+```
+
+Thanks to [Colin Bate](https://colinbate.com/) who contributed this feature.
+
+## Experimental: `astro:config` 
+
+Astro allows the users to retrieve some configuration information in various ways: the use of the environment variable `import.meta.env`, the `Astro` global object in `.astro` files, and the `context` variable in middleware/endepoints.
+
+Reading the same information sometimes felt confusing for beginners, for example reading `import.meta.env.BASE` or `Astro.base`.
+
+With `astro:config`, we want to provide a **unique** way to read some configuration options from anywhere inside the project, with type-safety in mind! The end goal would be to deprecate the properties, and make `astro:config` the de fact standard. Please leave your feedback in the relative [RFC](https://github.com/withastro/roadmap/pull/1106).
```

Comment:
Maybe worth mentionning it's not anywhere, it's within "vite's scope" (eg. not in an integration nor outside the astro project)

Reply:
```suggestion
With `astro:config`, we want to provide a *single* way to read the most useful configuration options from anywhere inside the project, with type-safety in mind! The end goal would be to deprecate the other ways of reading properties, and make `astro:config` the de facto standard. Please leave your feedback in the relevant [RFC](https://github.com/withastro/roadmap/pull/1106).
```